frameworks/CSharp/aspnetcore/PlatformBenchmarks/DateHeader.cs

```diff
@@ -27,7 +27,7 @@ internal static class DateHeader
 
     static DateHeader()
     {
-        var utf8 = Encoding.ASCII.GetBytes("\r\nDate: ").AsSpan();
+        var utf8 = "\r\nDate: "u8;
```

Comment:
```suggestion
        var utf8 = "\r\nDate: \r\n\r\n"u8;
```

And then we can get rid of lines 34-41 below.